hassio/src/dialogs/backup/show-dialog-backup-upload.ts

```diff
@@ -1,7 +1,9 @@
 import { fireEvent } from "../../../../src/common/dom/fire_event";
+import { Supervisor } from "../../../../src/data/supervisor/supervisor";
 import "./dialog-hassio-backup-upload";
 
 export interface HassioBackupUploadDialogParams {
+  supervisor?: Supervisor;
```

Comment:
Not needed

Reply:
Then we need to move/duplicate it to ui translations as well, as we do not have "Supervisor" access during onboarding

Reply:
Done